genie-web/src/integTest/java/com/netflix/genie/web/apis/rest/v3/controllers/JobRestControllerAgentExecutionIntegrationTest.java

```diff
@@ -27,7 +28,7 @@
  */
 @TestPropertySource(
     properties = {
-        JobRestController.AGENT_JOB_EXECUTION_KEY + "=true"
+        JobExecutionModeSelector.DEFAULT_EXECUTION_PROPERTY + "=true"
```

Comment:
This property name is confusing is this is to force agent execution.

Reply:
The property more clear: `genie.jobs.agent-execution.default`. 
But i agree this line (and the constant name) are very unclear.

Reply:
Renamed `DEFAULT_EXECUTE_WITH_AGENT_PROPERTY`